scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -432,19 +432,20 @@ def footprint_reports(self, report, show_footprint, all_deltas,
                            (report if not last_metrics else "the last twister run.")))
 
     def synopsis(self):
+        count = self.env.options.report_summary if self.env.options.report_summary else 10
```

Comment:
please use constant for default number in one place - at the command line option definitions for const and/or default.

Reply:
No, because it's two different default value. One for report_summary switch, one for whole twister.

Reply:
> No, because it's two different default value. One for report_summary switch, one for whole twister.

a command line option with its default set its mode for the 'whole twister', doesn't it ?